### PR TITLE
Issues-101 - Add step-by-step contributing guide

### PR DESCRIPTION
**Pull request by krisgesling**
Address #101 by adding a step-by-step contributor guide from signing the CLA through to submitting a PR (and helping review others :smile:).

Reviews, edits and additions warmly welcomed!

Is the main README the right place for this or should we split it out into a `CONTRIBUTING.md`?

### CODE REVIEW

**Comment by Tony763** [comment]
Nice one.
I would also add note to let functions named in English and just add language prefix.

`NUMBERS_FIXTURE_EN` to `NUMBERS_FIXTURE_CZ`

And also add notes to all non English texts for easier understanding and future reviews/editing.
```
    0.5: 'polovina',  #a half
    1.333: '1 a třetina', #1 and a third
    2.666: '2 a 2 třetiny' #2 and 2 thirds
```
With this even a non speaker can understood and work on functions.

File: readme.md [hunk 1]
```diff
@@ -199,3 +200,106 @@ testExtract("on the evening of june 5th 2017 remind me to call my mother",
...
+5. To check that everything is installed correctly, let's run the existing test-suite.
+```bash
+pytest
+```
+
+### 2. Writing tests
+We utilize a Test Driven Development (TDD) methodology so the first step is always to add tests for whatever you want to add or fix. If it's a bug, we must not have a test that covers that specific case so we want to add another test. If you are starting on a new language then you can take a look at the tests for other languages to get started.
+
+Tests are all located in `lingua_franca/test`.
+Each langauge should have two test files:
```
Comment (JarbasAl) [inline readme.md:250]: typo, language

**Comment by ChanceNCounter** [comment]
@krisgesling I made a couple changes and caught that typo. If you're good with it, I'm good.

**Comment by krisgesling** [comment]
Great, I'm sure this is just v1 and will evolve with the refactor.

**Pull request merged**